AdminWebsite/AdminWebsite/ClientApp/src/app/services/video-hearings.service.ts

```diff
@@ -441,4 +443,20 @@ export class VideoHearingsService {
     getJudge(): ParticipantModel {
         return this.modelHearing.participants.find(x => x.is_judge);
     }
+
+    isConferenceClosed(): boolean {
+        this.checkForExistingHearing();
+        return this.modelHearing.status === BookingStatus.Created && this.modelHearing.telephone_conference_id === '';
+    }
+
+    isHearingAboutToStart(): boolean {
+        this.checkForExistingHearing();
```

Comment:
As above

Reply:
Just because I think the following lines rely on whatever ^ that does completing...but I could be wrong